src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/VideoLinkBooking.kt

```diff
@@ -33,4 +33,10 @@ data class VideoLinkBooking(
   fun toAppointments(): List<VideoLinkAppointment> {
     return listOfNotNull(pre, main, post)
   }
+
+  fun matchesCourt(court: String?, courtId: String?): Boolean {
+    if (courtId != null) return main.courtId == courtId
+    if (court != null) return main.court == court
+    return true
```

Comment:
bit weird the default is true - will this evert realistically happen? could throw some form of assertion error

Reply:
(obviously it did use to default to true )

Reply:
This is how it currently works. If you don't specifiy a court (name) you get bookings for all courts.

Reply:
we'll be removing the all option as part of this work